README.md

```diff
@@ -147,7 +147,7 @@ make docker run_docker
 
 ## Community
 
-- Chat: [https://edgexfoundry.slack.com](https://join.slack.com/t/edgexfoundry/shared_invite/zt-65tadqv4-DtLS6eiz_AC7_FeJSdSm8A)
+- Chat: <https://slack.edgexfoundry.org/>
```

Comment:
Don't we want the join url since folks likely to use this will be new?

Reply:
New new link is a pretty UI.  The official EdgeX web site <https://www.edgexfoundry.org/community/> links here.  I figure it is best to route everyone through the same entrypoint.

Reply:
Ok